packages/protobuf/src/descriptors.ts

```diff
@@ -39,6 +39,73 @@ type SupportedFieldPresence = Extract<
   | FeatureSet_FieldPresence.LEGACY_REQUIRED
 >;
 
+/**
+ * Scalar value types. This is a subset of field types declared by protobuf
+ * enum google.protobuf.FieldDescriptorProto.Type The types GROUP and MESSAGE
+ * are omitted, but the numerical values are identical.
+ */
+export enum ScalarType {
```

Comment:
`ScalarType` and `LongType` move as well - they are closely tied to `DescField`, so they now live next to the definition of `DescField`, along with similar types.